File: Sayu/src/test/java/Array/ThreeSumTest.java
package Array;

import org.junit.jupiter.api.Test;

import java.sql.Array;
import java.util.ArrayList;
import java.util.List;

import static org.junit.jupiter.api.Assertions.*;

class ThreeSumTest {

    @Test
    public void  act(){
        List<List<Integer>> threeSum = new ThreeSum(new int[]{-1,0,1,2,-1,-4}).play();

        List<List<Integer>> expectedResult = new ArrayList<>();
        expectedResult.add(List.of(-1,-1,2));
        expectedResult.add(List.of(-1,0,1));
        assertEquals(threeSum, expectedResult);
    }

}

File: Algoexpert/src/test/kotlin/medium/ThreeSumTest.kt
package medium

import org.junit.Assert
import org.junit.Test


internal class ThreeSumTest {

    @Test
    fun threeNumberSum() {
        var finalR = ThreeSum().threeNumberSum(mutableListOf(12, 3, 1, 2, -6, 5, -8, 6),0)
        Assert.assertEquals(3,finalR.size)
    }
}